README.md

```diff
@@ -94,6 +94,7 @@ toast(options, onClose, onClick);
 -   `icon` - Override the default icon
 -   `color` - Override color with [semantic values](https://react.semantic-ui.com/collections/message/#variations-color)
 -   `size` - Size of toast with [semantic values](https://react.semantic-ui.com/collections/message/#variations-size)
+-   `list` - List of string for show item menu in toast
```

Comment:
I think the `list` explanation is a bit unclear and suggest replacing with something like :

`Array of strings for showing an item menu inside the toast`